src/Home.tsx

```diff
@@ -32,8 +32,17 @@ const Home = ({
 	const fileUploadMbLimit: number = 200 * 1024 * 1024;
 
 	const handleAllFiles = (files: any) => {
+		const unsupportedFiles: any[] = [];
 		for (let i = 0; i < files.length; i += 1) {
-			handleFile(files[i], i === (files.length - 1));
+			if (['application/javascript', 'text/javascript', 'application/xhtml+xml', 'text/html', 'text/htmlh'].indexOf(files[i].type) === -1) {
+				handleFile(files[i], i === (files.length - 1));
+			} else {
+				unsupportedFiles.push(files[i].name);
+			}
+		}
+
+		if (unsupportedFiles.length > 0) {
+			onModal('failed', `Selected file${unsupportedFiles.length > 1 ? 's' : ''} (${unsupportedFiles.join(', ')}) does not match our rules. We don't support executable files`);
```

Comment:
selected file(s) (...) can't be uploaded because of type restrictions (HTML and JS are forbidden)